xpano/log/logger.h

```diff
@@ -30,11 +30,16 @@ class Logger {
   const std::vector<std::string> &Log();
   void RedirectSpdlogOutput(std::optional<std::filesystem::path> app_data_path);
 
+  std::optional<std::string> GetLogFilePath();
+
  private:
   void Concatenate();
 
   std::vector<std::string> log_;
   std::shared_ptr<BufferSinkMt> sink_;
+
+  // to store log file path
```

Comment:
Please remove the comment. 

Also the naming should probably be different - we actually store a directory path here? E.g. `log_dir_` or `log_dir_path_` would be more appropriate.